src/main/java/frc/team997/lib/util/DashboardChooser.java

```diff
@@ -43,6 +45,9 @@
     private final Set<NetworkTableEntry> activeEntries = new LinkedHashSet<>();
     private final ReentrantLock mutex = new ReentrantLock();
 
+    private Logger<T> logger;
```

Comment:
final

Reply:
doesn't like this-- gives a compiler error "the blank final field logger may not have been initialized"